src/Browser/index.js

```diff
@@ -24,20 +27,31 @@ const AuthStates = require('./AuthStates');
  * @param {Object} [serviceCustomizations] override properties of serviceInfoUrl 
  * @returns {Pryv.Service}
  */
-async function setupAuth(settings, serviceInfoUrl, serviceCustomizations) {
-  return (new AuthController(settings, 
-    serviceInfoUrl, serviceCustomizations)).init();
+async function setupAuth (settings, serviceInfoUrl, serviceCustomizations, humanInteractionInterface) {
+  let store = getStore();
+  store.resetValues();
+  const authController = new AuthController(settings, serviceInfoUrl, serviceCustomizations);
+  const authService = await authController.init();
+  
+  if (settings.spanButtonID) {
+    let loginButton;
+    if (humanInteractionInterface == null) {
+      loginButton = new LoginButton(authController);
+    } else {
```

Comment:
how does this not break when  `humanInteractionInterface` is set to false? isn't the spanButtonID set?
If that's the case, maybe we should indicate it in the readme example https://github.com/pryv/lib-js/tree/refactor/modular-auth#authentication-process-for-virtual-dom-applications

Reply:
Good point, will mention, because the spanButtonID is not set. Actually it could be a good parameter for rendering or not rendering the UI. What do you think?

Reply:
I would prefer that, can you add an excerpt of authSettings in the README showing that it is unset?

Reply:
oki, sure :D